### PR TITLE
[15.0][MIG]purchase_stock_analytic: Migration to 15.0

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

**Comment by bizzappdev** [comment]
ready for review

**Comment by Saran440** [comment]
@bizzappdev Can you update code again, please?

I tested with `stock_analytic`. it's work. :+1:

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+odoo-addon-stock_analytic@ git+https://github.com/oca/account-analytic@refs/pull/448/head#subdirectory=setup/stock_analytic
```
Comment (Saran440) [inline test-requirements.txt]: stock_analytic is merged. Can you delete this file, please.

File: purchase_stock_analytic/tests/test_purchase_stock_analytic.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields
+from odoo.tests.common import SavepointCase
+
+
+class TestPurchaseStockAnalytic(SavepointCase):
```
Comment (Saran440) [inline purchase_stock_analytic/tests/test_purchase_stock_analytic.py]:
```suggestion
class TestPurchaseStockAnalytic(TransactionCase):
```

File: purchase_stock_analytic/tests/test_purchase_stock_analytic.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields
+from odoo.tests.common import SavepointCase
```
Comment (Saran440) [inline purchase_stock_analytic/tests/test_purchase_stock_analytic.py]:
```suggestion
from odoo.tests.common import TransactionCase
```

**Review by Saran440 — COMMENTED** [review]
Code reviewed.

Please, change your commit
`[WIP]purchase_stock_analytic: Added test-requirement]`
to
`[MIG] purchase_stock_analytic: Migration to 15.0`

**Review by Saran440 — APPROVED** [review]
Code Review 👍

File: purchase_stock_analytic/__manifest__.py [hunk 4]
```diff
@@ -5,7 +5,7 @@
     "name": "Purchase Stock Analytic",
     "summary": """
         Copies the analytic account of the purchase order item to the stock move""",
-    "version": "14.0.1.0.0",
+    "version": "15.0.0.1.0",
```
Comment (AaronHForgeFlow) [inline purchase_stock_analytic/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

**Review by AaronHForgeFlow — COMMENTED** [review]
Code LG!

File: purchase_stock_analytic/__manifest__.py (hunk 4 above)
Comment (bizzappdev) [reply purchase_stock_analytic/__manifest__.py]: Done

**Review by ps-tubtim — APPROVED** [review]
Functional Test: Ok

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

**Review by AaronHForgeFlow — APPROVED** [review]

**Pull request merged**